docs/release/trg-0/trg-8-04.md

```diff
@@ -0,0 +1,167 @@
+---
+title: TRG 8.04 - KIT Structure
+---
+
+| Status     | Created      | Post-History                           |
+|------------|--------------|----------------------------------------|
+| Draft      | 18-Jan-2024  | Initial contribution                   |
+
+## Project Structure
+
+Following our project structure, the collection of our KITs documentation is developed in the `./docs-kits/kits` folder, where each KIT is a subfolder called by its name for organisation purposes. The `Data Chain KIT`, for example, is defined here: `./docs-kits/kits/Data Chain Kit`.
+
+Each folder/KIT's content is structured in at least four pages/subfolders:
+
+- Adoption View -> `page_adoption-view.md`
+- Software Operation View -> `page_software-operation-view.md`
+- Documentation -> `page_documentation.md`
+- Software Development View/
+  - Specification -> `page_software-development-view.md`
+  - OpenAPI definition/
+  - Another OpenAPI definition/
+  - ...
+
+## Steps to add a NewKit documentation
```

Comment:
Also better to move to "KITs - Getting started".
This is also a bit too "low level" in my opinion. I think describing the necessary files is enough. Everyone contributing to `eclipse-tractusx` should now how to create directories and files.
And even if not, we are always happy to provide onboarding. 
"Click here.. type 'abc'.." kind of documentation seems to be unnecessarily verbose to me